### PR TITLE
July '24 Upstream Merge

### PR DESCRIPTION
**Pull request by whatston3**
## About the PR

Upstream merge.

## Why / Balance

Keeping up with the Joneses.

## How to test

Very carefully.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** ***this PR does not require an ingame showcase***

## Breaking changes

**Changelog**

No changelog for now, captured in upstream ones.

### CODE REVIEW

File: Resources/Prototypes/_NF/Entities/Clothing/Belt/belts_blood_cult.yml [hunk 1]
```diff
@@ -12,7 +12,6 @@
     sprite: _NF/Clothing/Belt/cult_force_field.rsi
   - type: Reflect
     enabled: true
-    reflectProb: .1
```
Comment (ErhardSteinhauer) [inline Resources/Prototypes/_NF/Entities/Clothing/Belt/belts_blood_cult.yml:15]: Is there a specific reason for removing this line?

Comment (ErhardSteinhauer) [reply Resources/Prototypes/_NF/Entities/Clothing/Belt/belts_blood_cult.yml:15]: Ah, I see what's going on. Ignore my question

**Comment by dvir001** [comment]
Going to have a fun time scanning this for new issues this coming weekend.

**Comment by Cheackraze** [comment]
looks good to me from my testing, im going to approve it with a reminder to do a Merge commit, _**not**_ a squash commit.

This involves a database migration so Im going to plan for this Monday to have this merged and I can do some brief server maintenance along with this.

**Review by Cheackraze — APPROVED** [review]

**Pull request merged**